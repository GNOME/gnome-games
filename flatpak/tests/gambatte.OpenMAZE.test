[Retro Reftest]
Path=/gambatte/OpenMAZE
Core=/app/lib/libretro/gambatte_libretro.so
Medias=OpenMAZE.gb;

[Frame 0]
