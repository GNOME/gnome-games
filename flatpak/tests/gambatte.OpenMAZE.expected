/gambatte/OpenMAZE/Boot: OK
/gambatte/OpenMAZE/Options: OK
/gambatte/OpenMAZE/0/FastForward: OK
/gambatte/OpenMAZE/0/Run: OK
